website/src/pages/faq.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from "react";
+import Layout from "@theme/Layout";
+import withBaseUrl from "@docusaurus/withBaseUrl";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import Link from "@docusaurus/Link";
+
+function Help(props) {
+	const context = useDocusaurusContext();
+	const { siteConfig = {} } = context;
+	const supportLinks = [
+		{
+			content: `Learn more using the [documentation on this site.](${withBaseUrl(
+				"docs/doc1"
+			)})`,
+			title: "Browse Docs"
+		},
+		{
+			content: "Ask questions about the documentation and project",
+			title: "Join the community"
+		},
+		{
+			content: "Find out what's new with this project",
+			title: "Stay up to date"
+		}
+	];
+	return (
+		<Layout title={siteConfig.title} description={siteConfig.description}>
+			<div className="docMainWrapper wrapper">
+				<div className="container padding-vert--lg padding-top--xl">
+					<div className="post">
+						<header className="postHeader">
```

Comment:
Use CSS modules approach. Refer to https://github.com/facebook/create-react-app/blob/master/docusaurus/website/src/pages/index.js